### PR TITLE
Use timestamps for the trajectories

### PR DESCRIPTION
**Pull request by toliver**

### CODE REVIEW

File: cyberglove_trajectory/src/cyberglove_trajectory_publisher.cpp [hunk 1]
```diff
@@ -308,11 +308,15 @@ const std::vector<std::string> CybergloveTrajectoryPublisher::glove_sensors_vect
       //Build and send the goal
 
       trajectory_goal_.trajectory.points.clear();
+      //WARNING if this node runs on a different machine from the trajectory controller, both machines will need to be synchronized
+      // chrony (sudo apt-get install crony) has been used successfully to achieve that
+      // The extra 10ms will allow time for the trajectory to get to the trajectory controller
+      trajectory_goal_.trajectory.header.stamp = ros::Time::now() + ros::Duration(0.010);
```
Comment (markpitchless) [inline cyberglove_trajectory/src/cyberglove_trajectory_publisher.cpp]: The 0.010 should really be a param.

Comment (toliver) [inline cyberglove_trajectory/src/cyberglove_trajectory_publisher.cpp]: Agree.

**Comment by ugocupcic** [comment]
Ahah ! That's what I was missing for https://github.com/shadow-robot/sr_moley/issues/141

**Comment by ugocupcic** [comment]
@shadowmanos can you try that on the glove then merge it? When you try it, can you record a quick bag for me as well? I'll then be able to send a pull request for https://github.com/shadow-robot/sr_moley/issues/141

**Pull request merged**